src/components/step/ConnectDeviceAccountStepContainer/ConnectDeviceAccountStepContainer.tsx

```diff
@@ -4,21 +4,16 @@ import ConnectDeviceAccountStep from "../ConnectDeviceAccountStep";
 import { getFitbitProviderID, getGarminProviderID, getOmronProviderID } from "../../../helpers/providerIDs";
 import { deviceType } from "../../../helpers/deviceType";
 
-export interface ConnectDeviceAccountStepContainerProps {
-    deviceType: deviceType
-}
 
-export default function (props: ConnectDeviceAccountStepContainerProps) {
+export default function () {
     const [title, setTitle] = useState<string>();
     const [text, setText] = useState<string>();
     const [styles, setStyles] = useState<any>({});
-    const [deviceType, setDeviceType] = useState<deviceType>(
-        props.deviceType
-    );
+    const [deviceType, setDeviceType] = useState<deviceType | undefined>(undefined);
```

Comment:
I think this can just be:

```const [deviceType, setDeviceType] = useState<deviceType>();```

By default it should already be a `deviceType | undefined` with a value of `undefined`.

Reply:
Huh, TIL